### ISSUE TITLE
npm install failing, unable to access a tarball

### ISSUE THREAD

**Issue by westandy** [openlayers/ol3#5535]
Command: npm install --save openlayers

npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
npm WARN deprecated wrench@1.5.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm ERR! fetch failed https://github.com/hegemonic/taffydb/tarball/7d100bcee0e997ee4977e273cdce60bd8933050e
npm WARN retry will retry, error on last attempt: Error: getaddrinfo ENOTFOUND github.com github.com:443
npm ERR! fetch failed https://github.com/hegemonic/taffydb/tarball/7d100bcee0e997ee4977e273cdce60bd8933050e
npm WARN retry will retry, error on last attempt: Error: getaddrinfo ENOTFOUND github.com github.com:443
npm ERR! fetch failed https://github.com/hegemonic/taffydb/tarball/7d100bcee0e997ee4977e273cdce60bd8933050e
npm ERR! Linux 3.10.0-327.10.1.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--save" "openlayers"
npm ERR! node v4.4.0
npm ERR! npm  v3.8.5
npm ERR! code ENOTFOUND
npm ERR! errno ENOTFOUND
npm ERR! syscall getaddrinfo

npm ERR! network getaddrinfo ENOTFOUND github.com github.com:443
npm ERR! network This is most likely not a problem with npm itself
npm ERR! network and is related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! Please include the following file with any support request:
npm ERR!     /*/npm-debug.log

[npm-debug.txt](https://github.com/openlayers/ol3/files/341959/npm-debug.txt)

**Comment by marcjansen** [comment]
It's right in the lines you copy and pasted:

> This is most likely not a problem with npm itself and is related to network connectivity.
> In most cases you are behind a proxy or have bad network settings.
>
> If you are behind a proxy, please make sure that the 'proxy' config is set properly. See: 'npm help config'

It just looks as if npm cannot access some dependency.

**Comment by westandy** [comment]
Right, it's complaining about some tarball off of taffydb.  I can get the link: https://github.com/hegemonic/taffydb/tarball/7d100bcee0e997ee4977e273cdce60bd8933050e through my browser.  So, I wasn't sure if it was the proxy or maybe pointing to a corrupt tarball, or something else.

**Comment by marcjansen** [comment]
You didn't bother at all. Hope you get it up.

**Comment by westandy** [comment]
I found out, that there's a dependency on jsdoc, and that is dependent on taffydb.  So, I went to jsdoc's github page, because sure enough, in it's package.json is not "~2.7.2", but the https link that my proxy was complaining about.  Since we are using sinopia on our side of the corporate firewall, I just changed the jsdoc package.json to point to ~2.7.2 and that part worked.  (I hope this makes sense.)
